### PR TITLE
Issue1164

### LINKED ISSUE #1164: Half life time should be nan if nuclide isn't found in database
When you try to retrieve the half life time of an unknown isotope 'inf' is returned. For example
`nucdat.half_life("Cm-253")` produces the output 'inf', while this isotope cannot be found in the nudat2 data https://www.nndc.bnl.gov/nudat2/chartNuc.jsp . 
Wouldn't it make more sense if pyne would return 'nan' in such cases, like it does for known isotopes with unknown half life time (e.g. U-241)?

### PR DESCRIPTION
**Pull request by ZoeRichter**
Answers issue #1164 and changes half_life in data.pyx to return nan instead of inf if the isotope cannot be found.  Previous behavior assumed nuclides not in data libraries are stable.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -134,6 +134,7 @@ v0.7.0 RC2
    * ENSDF database link to 2019 Oct 4th database
    * Update the C012-n.ace file link.
    * Missing elements name_to_zz dictionary
+   * Updated half_life in data.pyx to return nan if isotope not found
```
Comment (nsryan2) [inline CHANGELOG.rst]:
```suggestion
   * Updated half_life in data.pyx to return nan if isotope not found (#1257)
```

**Review by nsryan2 — COMMENTED** [review]
Other than adding the #PR to the changelog, LGTM!

**Comment by gonuke** [comment]
So... because of our rebasing workflow - you'll need to rebase this on the current version of the `0.7.0-rc` branch.  Is that something you're familiar with?  If not, we can walk you through it.

**Comment by ZoeRichter** [comment]
I'm not familiar with rebasing, no.

**Comment by ZoeRichter** [comment]
Based a conversation with @gonuke , there is a discussion about how to handle isotopes not in the library.
`from pyne import data
import numpy as np

#F19 is stable
print(data.half_life('F19'))
print(data.half_life('F19', use_metastable=False))
print(data.half_life('F19', use_metastable=True))
print()
#U235 is a known radioisotope
print(data.half_life('U235'))
print(data.half_life('U235', use_metastable=False))
print(data.half_life('U235', use_metastable=True))
print()
#Tb136 is a radioisotope with an unknown half-life
print(data.half_life('Tb136'))
print(data.half_life('Tb136', use_metastable=False))
print(data.half_life('Tb136', use_metastable=True))`

Is a quick example showing the three isotope cases we might see - known and radioactive, unknown and radioactive, and unknown and stable.  U235 outputs what you would expect, but calling data.half_life on F19 and Tb136 both return inf.  For all three isotopes. I saw no difference in output based on the use_metastable flag.

**Comment by gonuke** [comment]
I think the difference occurs if you pass in a metastable nuclide label, e.g. `Cf252m`

**Comment by gonuke** [comment]
Thanks for the contribution on this @ZoeRichter

**Pull request merged**